vllm/worker/model_runner.py

```diff
@@ -872,27 +909,93 @@ def capture_model(self, kv_caches: List[torch.Tensor]) -> None:
             bs for bs in _BATCH_SIZES_TO_CAPTURE if bs <= graph_batch_size
         ]
 
+        if self.attn_backend.get_name() == "flashinfer":
+            # For flashinfer, different batch sizes will share the
+            # same workspace buffer.
+            decode_workspace_buffer = \
+            torch.empty(FLASHINFER_WORKSPACE_BUFFER_SIZE,
+                                                dtype=torch.uint8,
+                                              device=self.device)
+            indices_buffer = torch.empty(self.cache_config.num_gpu_blocks,
```

Comment:
`self.cache_config.num_gpu_blocks` is too small as we may have duplicates. I think we should set it to `max_batch_size * self.cache_config.num_gpu_blocks`